src/floating-action-button.jsx

```diff
@@ -52,15 +52,16 @@ const FloatingActionButton = React.createClass({
     onTouchStart: React.PropTypes.func,
     secondary: React.PropTypes.bool,
     style: React.PropTypes.object,
+    zDepth: React.PropTypes.number,
```

Comment:
We have a more strict check for the zDepth. You can have a look at the `Paper` component.